pkg/apis/settings/settings.go

```diff
@@ -48,8 +50,10 @@ var defaultSettings = &Settings{
 
 // +k8s:deepcopy-gen=true
 type Settings struct {
+	AssumeRole                 string
 	ClusterName                string `validate:"required"`
 	ClusterEndpoint            string
+	CredsDuration              int `validate:"min=15"`
```

Comment:
You mark this as a duration but it's passed in in minutes. If you are going to use the term "duration", I think this should be a `time.Duration` value, so something like `10s` or `1h`

Reply:
Fixed in next revision.